client/src/App.js

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import logo from "./logo.svg";
 import "./style.css";
-
```

Comment:
To avoid conflicts, there should be no changes in this file

Reply:
Actually, I didn't add any code to App.js... if u find some conflicts, u can just delete..